repository_service_tuf_worker/repository.py

```diff
@@ -628,6 +633,34 @@ def bootstrap(
 
         return asdict(result)
 
+    def update_settings(
+        self,
+        payload: Dict[str, Any],
+        update_state: Optional[
+            Task.update_state
+        ] = None,  # It is required (see: app.py)
+    ):
+        """Update repository settings with the new settings."""
+        tuf_settings: Dict[str, Any] = payload.get("settings")
+        if tuf_settings is None:
+            raise KeyError("No 'settings' in the payload")
```

Comment:
nit: We should [stop raising exceptions to finish the task](https://github.com/repository-service-tuf/repository-service-tuf-worker/issues/315).
We should return the task with the details that `update_repository` as `False`